permissions/src/androidTest/java/com/google/accompanist/permissions/TestUtils.kt

```diff
@@ -127,21 +127,21 @@ internal fun doNotAskAgainPermissionInDialog(
 private fun UiObject.clickForPermission() {
     var objectExists = false
     var timesRetried = 0
-    while (!objectExists && timesRetried < 3) {
+    while (!objectExists && timesRetried < 5) {
```

Comment:
This might look better using an explicit max timeout. A bit like [`waitUntil()`](https://github.com/google/accompanist/blob/main/systemuicontroller/src/androidTest/java/com/google/accompanist/systemuicontroller/Utils.kt#L33)

Reply:
Fixed